index.html

```diff
@@ -1,223 +1,271 @@
+                <p>You can access your account from anywhere in the world on any device</p>
+                <a class="features-link-text" href=""><p class="text-features">Read More</p></a>
+              </div>
+            </li>
+            <li class="col col-4">
+              <img class="image-features" src="/images/coridor.jpg" alt="Glass coridor">
+              <div class="element-in-image">
+                <h3 class="title-small">Cloud Budget Management Software</h3>
+                <p>Our software is made so you can access and manage your budget</p>
+                <a class="features-link-text" href=""><p class="text-features">Read More</p></a>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="section-technology margin-70">
+        <div class="container">
+          <div class="row">
+            <div class="col col-2 offset-5">
+              <h2 class="title-medium text-center">Technology</h2>
+            </div>
+          </div>
+          <ul class="row text-center">
+            <li class="col col-3">
+              <p class="numbers">01</p>
+              <img class="image-technology" src="/images/cloud.svg" alt="Logo cloud">
+              <h3 class="title-small">Cloud Storage</h3>
+              <p>Access your account from anywhere in the world on any device</p>
+            </li>
+            <li class="col col-3">
+              <p class="numbers">02</p>
+              <img class="image-technology" src="/images/loop.svg" alt="Lofo loop">
+              <h3 class="title-small">Secure</h3>
+              <p>All your information is stored on secure cloud servers</p>
+            </li>
+            <li class="col col-3">
+              <p class="numbers">03</P>
+              <img class="image-technology" src="/images/pdf.svg" alt="Logo PDF">
+              <h3 class="title-small">PDF Download</h3>
+              <p>Download any of your reports in PDF format</p>
+            </li>
+            <li class="col col-3">
+              <p class="numbers">04</p>
+              <img class="image-technology" src="/images/csv.svg" alt="LOGO CSV">
+              <h3 class="title-small">CSV Download</h3>
+              <p>All your information is stored on secure cloud servers</p>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="margin-70">
+        <div class="container">
+          <div class="row">
+            <div class="col col-5">
+              <h2 class="title-medium">Stay focused on saving money</h2>
+              <p class="text-features">It is important to stay focused on saving money in any way you can. We help you monitor your spending habits so you can easily spot and cut any unnecessary expenses. Simply join today to get started!</p>
+              <a href="">
+                <img src="/images/playstore.svg" alt="Logo Playstore">
+              </a>

```

Comment:
Please, add aria-label for all the links and buttons without text inside. We need it for better a11y.